crates/python/src/executable.rs

```diff
@@ -250,6 +266,7 @@ impl PyExecutable {
                     translation_options,
                     execution_options.unwrap_or_default().as_inner(),
                 )
+                .with_current_context()
```

Comment:
I'm a little bit worried about these being easy to forget. It seems like there isn't any kind of warning or error raised if it's not there. I guess it just requires us to have a diligent eye when testing our traces.

Reply:
Yup, I think that's just the nature of the instrumentation library. To that end, I've implemented integration tests to cover the most important spans. If we refactor and drop one of these in a sensitive place, it will cause a test failure.